versioned_docs/version-22.10/releases/centreon-os.md

```diff
@@ -0,0 +1,141 @@
+  - Manual management of relationships between a user and ACL groups
+  - Automatic management of relationships between a user and ACL groups based on identity provider values
+- [Configuration] Improved the default Engine logger options for newly installed central servers and new pollers created
+  with the wizard
+- [Configuration] New Broker input/output stream types available, designed to support the new gRPC stream capability of
+  Broker, but that can also be used for the legacy BBDO over TCP protocol:
+  - BBDO Server: configures a server input or output, displaying only relevant fields for this purpose
+  - BBDO Client: configures a client input or output, displaying only relevant fields for this purpose
+- [Configuration] Some obsolete parameters have been removed from the Engine configuration menu
+- [Install] Added dependency management between modules during installation, update and deletion
+- [Install] Improved error handling during installation
+- [Install] Removed "Centreon Web Directory" parameter and use Apache configuration instead
+- [UI] Improved Centreon light and dark themes
+- [UI] Relabeled and improved tooltips for some fields in the Hosts, Host Templates, Host Groups, Services and Service
+  Templates configuration forms
+- [UI] Reworked the banner to be more responsive
+- [UX] Changing themes is now possible by clicking on the profile icon
+- [UX] The quick export button is now available by default for all users that have the admin privilege or the required
+  ACL action access. Deploying a configuration is now simpler for all users.
+- [Resources Status] Manage hosts and services categories (Detail tiles, Listing filter)
+- [Resources Status] Manage hosts and services severities (Logos, level, Detail tiles, Listing filter)
+- [Event logs] Rework CSV exporter in event logs to manage high volumetry
+- [Resources Status] Introduce CSV export in Timeline and Graphs
+- [Resources Status] New Parent Alias column and filter in listing for services
+- [Resources Status] Timeline UI in Resources Status cosmetic enhancements
+- [Anomaly Detection] New type of resources in Resource Status (display and filter)
+- [Anomaly Detection] Prediction enveloppe size management by user from Resources Status
+
+#### Performances
+
+- Move to PHP 8.1
+- The "conf changed" flag in the Poller configuration menu now relies on a faster database query, improving the time to display the page
+
+#### Breaking changes
+
+> The configuration of authorizations via the OpenID Connect protocol has evolved. Automatic addition to a contact
+> group and role management have been improved. It is necessary to review your OpenID Connect configuration.
+
+> In /monitoring/resources API, ‘severity_level’ api return key has been replaced by a json object ‘severity’ containing id, level, name and icon
+
+
+
+## Centreon Collect
+
+### 22.10.0
+
+#### Centreon Engine
+
+##### Enhancements
+
+###### Anomaly Detection improvements
```

Comment:
Title to remove and add prefixes instead on each item: [Anomaly Detection] or [Ano-Detection]?